lib/fluent/plugin/out_lm.rb

```diff
@@ -28,6 +29,8 @@ class LmOutput < BufferedOutput
 		
     config_param :force_encoding,  :string, :default => ""
 
+    config_param :compression,  :string, :default => ""
```

Comment:
Any reason to not enable gzip by default?

Reply:
Not any, just to have an option to not gzip it, if the user wants to.

Reply:
And also, We have not tested yet! how much increase in cpu would it cost for k8s integration.

Reply:
As we discussed during standup - I agree, let’s keep it off by default until we have tested the impact in k8s.